scripts/appverifier_xml.py

```diff
@@ -0,0 +1,272 @@
+    },
+    "SRWLock": {
+        "0x250": "A thread tried to use SRW lock that is not initalized",
+        "0x251": "The SRW lock is being re-initialized",
+        "0x252": "The SRW lock is being released with a wrong release API",
+        "0x253": "The SRW lock is being acquired recursively by the same thread",
+        "0x254": "The thread that owns the SRW lock is exiting or being terminated",
+        "0x255": "The SRW lock is being released by the thread that did not acquire the lock",
+        "0x256": "The memory address being freed contains an active SRW lock that is still in use",
+        "0x257": "The DLL being unloaded contains an active SRW lock that is still in use"
+    },
+    "Threadpool": {
+        "0x700": "Thread priority is changed when thread is returned to threadpool",
+        "0x701": "Thread affinity is changed when thread is returned to threadpool",
+        "0x702": "One or more messages left as unprocessed when threadpool thread is returned to the threadpool",
+        "0x703": "Any window is kept alive when threadpool thread is returned to the threadpool",
+        "0x704": "ExitThread is called on a threadpool thread",
+        "0x705": "Callback function changed the thread token to impersonate another user and forgot to reset it before "
+        "returning it to the threadpool",
+        "0x706": "Windows API that requires dedicated or persistent thread called from threadpool",
+        "0x707": "Callback function forgot to close or reset the current transaction handle",
+        "0x708": "Callback function called CoInit and CoUnInit in differing amounts (unbalanced)",
+        "0x709": "The period to signal the timer is not zero when the timer is set to signal only once with the WT_EXECUTEONLYONCE flag",
+        "0x70A": "The loader lock is held within the callback and is not released when the thread is returned to the threadpool",
+        "0x70B": "The preferred language is set within the callback and is not cleared when the thread is returned to the threadpool",
+        "0x70C": "The background priority is set within the callback and is not disabled when the thread is returned to the threadpool",
+        "0x70D": "TerminateThread called on a threadpool thread",
+    },
+    "TLS": {
+        "0x350": "A DLL that allocated a TLS index is being unloaded before freeing that TLS index",
+        "0x351": "The internal verifier structures used to store the state of TLS slots for thread are corrupted",
+        "0x352": "An invalid TLS index is used"
+    }
+}
+
+
+def parseXML(filepath, dump_xml_on_error):
+    xml_is_app_verifier = False
+    app_verifier_entries = []
+
+    print("Looking for AppVerifier XML file...", flush=True)
+
+    try:
+        xml_tree = ElementTree.parse(filepath)
+    except Exception: # lgtm [py/catch-base-exception]
+        print("ERROR: Exception occured while trying to open XML file!", flush=True)
+        return -1
```

Comment:
trivial: why bother trying to catch an exception only to kill the program? just let the exception happen